daemon/src/engine/framework/ConsoleHistory.h

```diff
@@ -31,18 +31,40 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef FRAMEWORK_CONSOLE_HISTORY_H_
 #define FRAMEWORK_CONSOLE_HISTORY_H_
 
+#include <string>
+#include <vector>
+#include <mutex>
```

Comment:
So should I include Common.h?
ConsoleHistory.cpp had
```
#include "qcommon/q_shared.h"
#include "qcommon/qcommon.h"
```
Though only stuff from q_shared.h would be needed